.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Release
+title: "[RELEASE] Release version {{ env.VERSION }}"
+labels: untriaged, release
+---
+
+## Release OpenSearch and OpenSearch Dashboards {{ env.VERSION }}
+
+I noticed that a manifest was automatically created in [manifests/{{ env.VERSION }}](/opensearch-project/opensearch-build/tree/main/manifests/{{ env.VERSION }}). Please follow the following checklist to make a release.
+
+### Preparation
+
+- [ ] Assign this issue to a release owner.
+- [ ] Document any new quality requirements or changes.
+- [ ] Declare a pencils down date for new features to be merged.
+- [ ] Finalize scope and feature set and update [the Public Roadmap](https://github.com/orgs/opensearch-project/projects/1).
+- [ ] [Create a version label](https://github.com/opensearch-project/opensearch-plugins/blob/main/META.md#create-or-update-labels-in-all-plugin-repos) in each component repo.
+- [ ] [Create a release issue in every component repo](https://github.com/opensearch-project/opensearch-plugins/blob/main/META.md#create-an-issue-in-all-plugin-repos) that links back to this issue.
+- [ ] Ensure that all release issues created above are assigned to an owner in the component team.
+
+### CI/CD
+
+- [ ] Create Jenkins workflows that run daily snapshot builds for OpenSearch and OpenSearch Dashboards. 
+- [ ] Increment each component version to {{ env.VERSION }} and ensure working CI in component repositories.
+- [ ] Make pull requests to add each component to [manifests/{{ env.VERSION }}/opensearch-{{ env.VERSION }}.yml](/opensearch-project/opensearch-build/tree/main/manifests/{{ env.VERSION }}/opensearch-{{ env.VERSION }}.yml) and [manifests/{{ env.VERSION }}/opensearch-dashboards-{{ env.VERSION }}.yml](/opensearch-project/opensearch-build/tree/main/manifests/{{ env.VERSION }}/opensearch-dashboards-{{ env.VERSION }}.yml) with the corresponding checks.
+
```

Comment:
In this section or as part of the release section

---

Verify all {{ env.VERSION }} labeled issues have been triaged and resolved.

Reply:
adding